TWLight/users/views.py

```diff
@@ -756,7 +757,7 @@ def get_context_data(self, **kwargs):
                                         Application.QUESTION,
                                         Application.APPROVED,
                                     ),
-                                    partner=each_authorization.partners,
+                                    partner=each_authorization.partners.first(),
```

Comment:
seems like this should be `.get()` instead of first so we raise an exception if there is more than one result? Maybe we need to split out some of this to its own method function to allow separate handing for bundle